IPython/core/tests/test_oinspect.py

```diff
@@ -282,7 +281,7 @@ def test_info():
     nt.assert_equal(i['docstring'], Call.__doc__)
     nt.assert_equal(i['source'], None)
     nt.assert_true(i['isclass'])
-    nt.assert_equal(i['init_definition'], "Call(self, x, y=1)\n")
+    nt.assert_equal(i['init_definition'], "Call(x, y=1)\n")
```

Comment:
Intriguing! I guess it gets a different result on py2. My first thought is to let it differ and handle it in the test. Or do you think I should see if I can fix the fact that signatures include self on py2?